src/test/fixtures/NonExtendingSpan.ts

```diff
@@ -0,0 +1,85 @@
+import * as noop from '../../noop';
+import SpanContext from '../../span_context';
+import Tracer from '../../tracer';
+
+/**
+ * Span implementation for unit tests. Does not extend opentracing.Span, but
+ * implements the interface
+ */
+export class NonExtendingSpan {
```

Comment:
this seems a bit of an overkill to test one check, it could've been done with just an anonymous dict. Why maintain all this boilerplate that doesn't add anything useful?

Reply:
Thanks for the helpful suggestions, I got rid of the fixtures indeed :)